docs/diagrams/wallet-lifecycle/wallet-lifecycle-2-entering-moving-funds-state.tikz

```diff
@@ -0,0 +1,182 @@
+\draw[->]      (rt_check7)   -| (rt_action10) \no;
+\draw[->]      (rt_action9)  |- (rt_action10);
+\draw[->]      (rt_action10) -- (rt_action11);
+
+
+% CLOSEABLE WALLET
+% Nodes
+\node[start, right=of rt_start, xshift=210mm]                    (cw_start)   {Somebody notifies about \textbf{closeable wallet}};
+\node[check, below=of cw_start]                                  (cw_check1)  {Is wallet marked\\ as ACTIVE?};
+\node[check, below left=of cw_check1]                            (cw_check2)  {Is wallet LIVE?};
+\node[revert, below right=of cw_check1]                          (cw_revert1) {Transaction reverted};
+\node[check, below right=of cw_check2]                           (cw_check3)  {Is wallet\\ older than\\ wallet\_max\_age or\\ is wallet's balance below\\ wallet\_min\_closure\_btc?};
+\node[revert, below left=of cw_check3, xshift=-2cm]              (cw_revert2) {Transaction reverted};
+\node[note1, right=of cw_start.south east, anchor=south west]    (cw_note1)   {Can be successfully called for wallets:\\
+                                                                               \textbf{- LIVE}};
+\node[note2, above=of cw_note1.north, anchor=south, yshift=-8mm] (cw_note2)   {Sender incentivized by:\\ \textbf{Reimbursement, if maintainer}};
+
+% Arrows
+\draw[->] (cw_start)  -- (cw_check1);
+\draw[->] (cw_check1) -| (cw_check2)  \no;
+\draw[->] (cw_check1) -| (cw_revert1) \yes;
+\draw[->] (cw_check2) -| (cw_revert2) \no;
+\draw[->] (cw_check2) -| (cw_check3)  \yes;
+\draw[->] (cw_check3) -| (cw_revert2) \no;
+\draw[->] (cw_check3) -| (rt_check6)  \yes;
+
+
+% WALLET HEARTBEAT FAILED
+% Nodes
+\node[start, right=of cw_start, xshift=140mm]                     (whf_start)   {Somebody notifies about \textbf{wallet failing heartbeat}};
+\node[check, below=of whf_start]                                  (whf_check1)  {Is sender a\\ wallet member?};
+\node[check, below right=of whf_check1]                           (whf_check2)  {Is the\\ notification\\ supported by at\\ least 51 valid\\ signatures?};
+\node[check, below right=of whf_check2]                           (whf_check3)  {Is wallet LIVE?};
+\node[revert, below left=of whf_check3, xshift=-6cm]              (whf_revert1) {Transaction reverted};
+\node[action, below right=of whf_check3]                          (whf_action1) {Mark inactive operators as ineligible for rewards};
+\node[note1, right=of whf_start.south east, anchor=south west]    (whf_note1)   {Can be successfully called for wallets:\\
+                                                                                 \textbf{- LIVE\\
+                                                                                 - MOVING\_FUNDS\\
+                                                                                 - CLOSING}};
+\node[note2, above=of whf_note1.north, anchor=south, yshift=-8mm] (whf_note2)   {Sender incentivized by:\\ \textbf{Fear of loosing rewards for 2 weeks; reimbursement}};
```

Comment:
If the wallet dies, caller may get slashed. It's not about losing rewards from 2 weeks.

```suggestion
\node[note2, above=of whf_note1.north, anchor=south, yshift=-8mm] (whf_note2)   {Sender incentivized by:\\ \textbf{Ether reimbursed, closing a problematic wallet to avoid slashing}};
```